opencti-platform/opencti-front/src/private/components/events/stix_sighting_relationships/StixSightingRelationshipEdition.jsx

```diff
@@ -10,6 +10,17 @@ import useQueryLoading from '../../../../utils/hooks/useQueryLoading';
 
 const useStyles = makeStyles((theme) => ({
   drawerPaper: {
+    minHeight: '100vh',
```

Comment:
The animation is missing on both screen (graph and not graph)

Reply:
I don't understand which animation you're talking about

Reply:
I can see that
It's a problem on my side
All good now, sorry

Reply:
what's strange is that I have been able to reproduce the issue